src/olympia/files/utils.py

```diff
@@ -805,6 +806,45 @@ def update_version_number(file_obj, new_version_number):
     shutil.move(updated, file_obj.file_path)
 
 
+def write_crx_file(chunks, storage, loc):
```

Comment:
Actually, ignore that, given the way the method is used I like `write_crx_file` I think.